### PR TITLE
Code and comments style

### PR DESCRIPTION
**Pull request by grandizzy**
More readable reward bucket take function

# Description of change
## High level
* Code and comments maintenance:
  * improve readability of `_rewardBucketTake` function: declare `takerLPReward` (inline with `kickerLPReward`) and use it instead mutating `totalLPReward`. Check if `totalLPReward != 0` in order to save `LP` in storage and use `takerLPReward` in `BucketTakeLPAwarded` event (instead doing another operation `totalLPReward - kickerLPReward`)
  * consistent code comments, all above the line of code but special mentions comment at the right of same line

# Description of bug or vulnerability and solution
* Code and comments maintenance. No change in logic
  * improve readability of `_rewardBucketTake` function
  * consistent comments style

# Contract size
## Pre Change
```
TakerActions             -  15,189B  (61.80%)
```
## Post Change
```
TakerActions             -  15,191B  (61.81%)
```

# Gas usage
## Pre Change
```
ERC20  :  | bucketTake                           | 8438            | 119944 | 127944 | 399161 | 35      |
ERC721: | bucketTake                             | 77563           | 282300 | 192808 | 545748   | 14      |
```
## Post Change
```
ERC20  :  | bucketTake                           | 8438            | 119170 | 127914 | 375732 | 35      |
ERC721: | bucketTake                             | 77539           | 282239 | 192663 | 545718   | 14      |
```

### CODE REVIEW

**Review by EdNoepel — APPROVED** [review]
I'm not a fan of moving annotation comments above the line.  It complicates readability of the code itself, as the reader has more to process.  Approving anyway for improved readability of `TakerActions.kickWithDeposit`.

File: src/libraries/external/LenderActions.sol [hunk 1]
```diff
@@ -506,7 +507,7 @@ library LenderActions {
         } else {
             // update lender LP balance
```
Comment (prat-gpt) [inline src/libraries/external/LenderActions.sol]: Can be updated to `// update lender and bucket LP balances` like https://github.com/ajna-finance/contracts/pull/779/files#diff-58912675070028eef46d84432f10430f775be631c835f44690c9d15b494da2c4R428 and remove the comment below for consistency

File: src/libraries/external/LenderActions.sol [hunk 2]
```diff
@@ -685,7 +688,7 @@ library LenderActions {
...
```
Comment (prat-gpt) [inline src/libraries/external/LenderActions.sol]: Can be updated to `// update lender and bucket LP balances` like https://github.com/ajna-finance/contracts/pull/779/files#diff-58912675070028eef46d84432f10430f775be631c835f44690c9d15b494da2c4R428 and remove the comment below for consistency

File: src/libraries/external/TakerActions.sol [hunk 3]
```diff
@@ -608,29 +613,30 @@ library TakerActions {
...
         // if arb take - taker is awarded collateral * (bucket price - auction price) worth (in quote token terms) units of LPB in the bucket
         if (!depositTake_) {
-            uint256 takerReward = Maths.wmul(vars.collateralAmount, vars.bucketPrice - vars.auctionPrice);
-
-            totalLPReward = Maths.wdiv(takerReward, exchangeRate);
+            takerLPReward = Maths.wdiv(
+                Maths.wmul(vars.collateralAmount, vars.bucketPrice - vars.auctionPrice),
+                exchangeRate
+            );
+            totalLPReward += takerLPReward;
```
Comment (mattcushman) [inline src/libraries/external/TakerActions.sol]: `totalLPReward` is always zero here, so why `+=`?

**Review by prat-gpt — APPROVED** [review]
LGTM! Just added a comment suggestion

File: src/libraries/external/TakerActions.sol [hunk 4]
```diff
@@ -641,24 +647,24 @@ library TakerActions {
...
-
+        if (totalLPReward != 0) bucket.lps += totalLPReward;
         // collateral is added to the bucket’s claimable collateral
         bucket.collateral += vars.collateralAmount;
 
         emit BucketTakeLPAwarded(
             msg.sender,
             vars.kicker,
-            totalLPReward - kickerLPReward,
+            takerLPReward,
```
Comment (mattcushman) [inline src/libraries/external/TakerActions.sol:667]: this is nicer 👍

**Review by mattcushman — APPROVED** [review]
lgtm.  I think the code is fine, one minor style question added in comment

File: src/libraries/external/LenderActions.sol (hunk 1 above)
Comment (grandizzy) [reply src/libraries/external/LenderActions.sol]: yep, changed with https://github.com/ajna-finance/contracts/pull/779/commits/a78c531a805c3fb86e0819508da5ac0401045962

File: src/libraries/external/LenderActions.sol (hunk 2 above)
Comment (grandizzy) [reply src/libraries/external/LenderActions.sol]: changed with https://github.com/ajna-finance/contracts/pull/779/commits/a78c531a805c3fb86e0819508da5ac0401045962

File: src/libraries/external/TakerActions.sol (hunk 3 above)
Comment (grandizzy) [reply src/libraries/external/TakerActions.sol]: good point, changed in https://github.com/ajna-finance/contracts/pull/779/commits/a78c531a805c3fb86e0819508da5ac0401045962

**Pull request merged**